src/_partials/_head.erb

```diff
@@ -31,4 +31,7 @@
 <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.0/languages/json.min.js"></script>
 <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.0/languages/xml.min.js"></script>
 
+<%# google material icon %>
```

Comment:
what's this being used for? I'm not excited about pulling in a Google webfont for a hamburger menu icon.

Reply:
it is use to get 'hamburger menu' and 'close icon' sir,
ok, thank you for review i'll add the svg , of the hamburger menu and close icon